code/modules/unit_tests/asset_smart_cache.dm

```diff
@@ -0,0 +1,65 @@
+/datum/asset/spritesheet_batched/test
+	name = "test"
+	load_immediately = TRUE
+	force_cache = TRUE
+	// Don't let the asset subsystem load this. This is how we trick it.
+	_abstract = /datum/asset/spritesheet_batched/test
+	var/static/list/items = list(/obj/item/binoculars, /obj/item/camera, /obj/item/clothing/under/color/blue, /obj/item/clothing/under/color/black)
+
+/datum/asset/spritesheet_batched/test/create_spritesheets()
+	for(var/atom/item as() in items)
```

Comment:
`for(var/atom/item as anything in items)` should work too, unless there is something more to this syntax I'm unaware of.

Reply:
yeah please do not use as()